.github/workflows/build-and-dockerize.yaml

```diff
@@ -55,6 +55,10 @@ jobs:
         if: ${{ inputs.build }}
         run: pnpm build
 
+      - name: CLI prepack
+        if: ${{ inputs.build }}
+        run: pnpm run --filter @graphql-hive/cli workspace-fix
```

Comment:
I had to add this step, otherwise, the CLI building fails for the alpha release - however, it does not fail for stable, which uses this same workflow 😕